tests/itest_specs/basic_round_trip/long_put.cairo

```diff
@@ -314,9 +314,9 @@ namespace LongPutRoundTrip {
         ///////////////////////////////////////////////////
 
         %{
-            stop_mock_oracle_1()
-            stop_mock_oracle_2 = mock_call(
-                ids.EMPIRIC_ORACLE_ADDRESS, "get_value", [1450000000000000000000, 18, 0, 0]  # mock current ETH price at 1450
+            stop_mock_current_price()
+            stop_mock_current_price_2 = mock_call(
+                ids.tmp_address, "get_spot_median", [1450000000000000000000, 18, 0, 0]  # mock current ETH price at 1450
```

Comment:
does the get_value not work anymore in the new contract? asking because this potentially causes a non-compatibility risk if empiric drops endpoint on their upgradable contract...

Reply:
The endpoint is there, but I could not get it to return anything but zeros,

Reply:
Well the endpooint is there, but whenever i tried to use it, it just returned zeros